eZ/Bundle/EzPublishLegacyBundle/Cache/SwitchableHttpCachePurger.php

```diff
@@ -43,4 +43,14 @@ public function purgeAll()
 
         $this->gatewayCachePurger->purgeAll();
     }
+
+    public function purgeForContent( $contentId )
```

Comment:
ok... is just there are phpdocs in repository code for classes implementing functions too...
https://github.com/ezsystems/ezpublish-kernel/blob/master/eZ/Publish/API/Repository/ContentService.php#L45
https://github.com/ezsystems/ezpublish-kernel/blob/master/eZ/Publish/Core/Repository/ContentService.php#L137